extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -1,66 +1,66 @@
 /*global define*/
 /*jshint maxlen:125, camelcase:false, maxdepth:7*/
 define('ext.wikia.adEngine.provider.gpt.googleTag', [
+	'ext.wikia.adEngine.provider.gpt.googleSlots',
 	'ext.wikia.aRecoveryEngine.recovery.helper',
+	'ext.wikia.adEngine.adLogicPageViewCounter',
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (helper, doc, log, window) {
+], function (googleSlots, helper, adLogicPageViewCounter, doc, log, window) {
 	'use strict';
 
 	var logGroup = 'ext.wikia.adEngine.provider.gpt.googleTag',
 		registeredCallbacks = {},
-		slots = {},
 		slotQueue = [],
 		pageLevelParams,
-		pubAds;
+		initialized = false;
 
-	function GoogleTag() {
```

Comment:
After talking with @fraszczakszymon we decided to not use the `prototype` chain to make this module consistent with the rest of AdEngine JS modules.